drupal/settings/production.settings.php

```diff
@@ -49,3 +49,8 @@
 if (is_numeric($file_gc = getenv('GOVCMS_FILE_TEMP_MAX_AGE'))) {
   $config['system.file']['temporary_maximum_age'] = $file_gc;
 }
+
+$config['lagoon_logs.settings']['host'] = 'application-logs.lagoon.svc';
+$config['lagoon_logs.settings']['port'] = 5140;
+$config['lagoon_logs.settings']['identifier'] = 'drupal';
+$config['lagoon_logs.settings']['enable'] = TRUE;
```

Comment:
Just reviewing this; it looks like this may have been reverted in lagoon logs.

https://git.drupalcode.org/project/lagoon_logs/-/blob/2.x/src/Logger/LagoonLogsLoggerFactory.php#L25